### PR TITLE
Updated badged to look more modern and unified

### PR DESCRIPTION
**Pull request by bramceulemans**
See title. 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
+[![GitHub Workflow Status](https://img.shields.io/github/workflow/status/FakerPHP/Faker/Continuous%20Integration/main)](https://github.com/FakerPHP/Faker/actions)
+[![StyleCI](https://github.styleci.io/repos/307658860/shield?branch=main&style=flat)](https://github.styleci.io/repos/307658860?branch=main)
 
 Faker is a PHP library that generates fake data for you. Whether you need to bootstrap your database, create good-looking XML documents, fill-in your persistence to stress test it, or anonymize data taken from a production service, Faker is for you.
 
 Faker is heavily inspired by Perl's [Data::Faker](http://search.cpan.org/~jasonk/Data-Faker-0.07/), and by ruby's [Faker](https://rubygems.org/gems/faker).
 
 Faker requires PHP >= 7.1.
 
-
```
Comment (GrahamCampbell) [inline README.md:13]: Don't delete this.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bramceulemans) [reply README.md:13]: You mean the dangling dash?

Comment (bramceulemans) [reply README.md:13]: Did not realise Markdown interprets a double new line different from a single one.

**Comment by GrahamCampbell** [comment]
Thanks.

**Pull request merged**